c_src/eleveldb.cc

```diff
@@ -568,8 +613,13 @@ async_open(
     eleveldb_priv_data& priv = *static_cast<eleveldb_priv_data *>(enif_priv_data(env));
 
     leveldb::Options *opts = new leveldb::Options;
+    opts->expiry_module.assign(priv.m_ExpiryModule.get());
+
     fold(env, argv[2], parse_open_option, *opts);
+
     opts->fadvise_willneed = priv.m_Opts.m_FadviseWillNeed;
+    if (NULL==priv.m_ExpiryModule.get() && NULL!=opts->expiry_module.get())
```

Comment:
I am unclear on the purpose of this code. A few lines above, you assigned opts->expiry_module from priv.m_ExpiryModule, so they should be the same. If there's some reason why they may not be the same, a comment here would be helpful.